core/templates/pages/classroom-page/classroom-page.component.css

```diff
@@ -159,6 +154,9 @@ classroom-page .oppia-classroom-page-subheading-text {
     margin-top: -7vh;
     width: 50vw;
   }
+  .oppia-classroom-banner img {
```

Comment:
Please use a class name here instead of img or use `w-100` [boostrap class](https://getbootstrap.com/docs/4.0/utilities/sizing/) in the HTML.